python/py-panedr/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-panedr
+version             0.5.2
+revision            0
+categories-append   science
+platforms           darwin
+license             GPL-2.0+
+
+python.versions     27 36 37 38
+
+maintainers         {gmail.com:giovanni.bussi @GiovanniBussi} openmaintainer
+
+description         Panedr reads a Gromacs EDR file and returns its content as a pandas dataframe.
+long_description    ${description} \
+                    The library exposes one function—the edr_to_df function—that gets the path \
+                    to an EDR file and returns a pandas dataframe.
+
+homepage            https://github.com/jbarnoud/panedr
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+
+distname            ${python.rootname}-${version}
+
+checksums           rmd160  aab8bda18f35abdd33bde44e962c5b0c5cef2314 \
+                    sha256  45eb926a1176789998f041be4d2bea2ed5b70c6540cbaf8903d82473cfd886b2 \
+                    size    15642174
+
+if {${name} ne ${subport}} {
+    depends_build-append    port:py${python.version}-setuptools \
+                            port:py${python.version}-pbr
+    depends_lib-append      port:py${python.version}-pandas
```

Comment:
since the PyPI package includes the unittests, it's worth considering to add a `test.phase`. In this case it would be like this:
```
    depends_test-append \
                    port:py${python.version}-pytest \
                    port:py${python.version}-six

    test.run        yes
    test.cmd        py.test-${python.branch}
    test.target
    test.env        PYTHONPATH=${worksrcpath}/build/lib
```

Reply:
I thought installation from github was needed (and that was not working for me). But you are right, the pypi package includes tests, and your suggestions works perfectly. Thanks a lot!